modules/apps/forms-and-workflow/polls/polls-service/src/main/java/com/liferay/polls/internal/verify/PollsServiceVerifyProcess.java

```diff
@@ -54,6 +56,13 @@ protected void verifyAuditedModels() throws Exception {
 		}
 	}
 
+	protected void verifyGroupedModels() throws Exception {
```

Comment:
Hi @leoadb, can you **verify** if this changes are ok in `polls-service`? (Or point us to who is in charge of it?

Thanks!